bootstrap/src/main/java/com/lightbend/lagom/javadsl/akka/bootstrap/Bootstrap.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (C) 2016-2018 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package com.lightbend.lagom.javadsl.akka.bootstrap;
+
+import akka.actor.ActorSystem;
+import akka.actor.ExtendedActorSystem;
+import akka.management.AkkaManagement;
+import akka.management.cluster.bootstrap.ClusterBootstrap;
+import com.typesafe.config.Config;
+
+import javax.inject.Inject;
+import javax.inject.Singleton;
+
+@Singleton
+public class Bootstrap {
+
+    @Inject
+    public Bootstrap(ActorSystem actorSystem) {
+        Config config = actorSystem.settings().config().getConfig("lagom.akka.bootstrap");
+        if (config.getBoolean("start-akka-management")) {
+            new AkkaManagement((ExtendedActorSystem) actorSystem).start();
+        }
+        if (config.getBoolean("start-cluster-bootstrap")) {
+            new ClusterBootstrap((ExtendedActorSystem) actorSystem).start();
+        }
+    }
```

Comment:
I have left a comment before, but due some unhandy manipulation of my part, I finished by deleting my review. 

So, my previous comment / question was: 

`ClusterBootstrap` requires `AkkaManagement`. The only reason of starting it without `AkkaManagement` would be if we want to add the Cluster HTTP endpoint directly as an additional router in Lagom. Is that the reason why you are making this configurable?